src/__tests__/runner.test.js

```diff
@@ -323,15 +334,31 @@ describe('Runner', () => {
       expect(spawn).toBeCalledWith('taskkill', ['/pid', '123', '/T', '/F']);
     });
 
-    it('calls kill() to close the process on POSIX', () => {
+    it('calls process.kill() with processGroup id to close the process on POSIX', () => {
+      const workspace: any = {};
+      const sut = new Runner(workspace);
+      process.platform = 'posix';
+      const kill = jest.fn();
+      sut.debugprocess = ({kill, pid: 123}: any);
+      sut.closeProcess();
+
+      expect(kill).not.toBeCalled();
```

Comment:
Non-blocking: This is another test we should **eventually** clean up to communicate the behaviour we're expecting. If this failed, I wouldn't know what was wrong unless I looked at the function under test to see that `processKill` somehow threw an error. (Like the next test 😁).